src/onegov/search/integration.py

```diff
@@ -375,24 +408,64 @@ def reindex_model(model):
                     q = q.filter(i.polymorphic_on == i.polymorphic_identity)
 
                 for obj in q:
-                    self.es_orm_events.index(self.schema, obj)
+                    self.es_orm_events.index(schema, obj)
+
+            except Exception as e:
+                print(f'Error psql indexing model \'{model}\': {e}')
             finally:
                 session.invalidate()
                 session.bind.dispose()
 
-        # by loading models in threads we can speed up the whole process
+        models = self.get_searchable_models()
+        index_log.info(f'Number of models to be indexed: {len(models)}')
+
         with ThreadPoolExecutor() as executor:
             results = executor.map(
-                reindex_model, (
-                    model
-                    for base in self.session_manager.bases
-                    for model in searchable_sqlalchemy_models(base)
-                )
+                reindex_model, (model for model in models)
             )
             if fail:
-                tuple(results)
+                print(tuple(results))
 
         self.es_indexer.bulk_process()
+        self.psql_indexer.bulk_process()
+
+        index_log.info('Done')
+
+    def psql_index_status(self):
+        """ Prints how many document are indexed per model. """
+
+        success = True
+        models = self.get_searchable_models()
+
+        print(f'Check psql indexing status, schema {self.schema} ..')
+        session = self.session()
+        for model in models:
+            try:
+                q = session.query(model.fts_idx)
+                total = q.count()
+                if total == 0:
+                    continue  # empty table
+            except Exception as e:
+                print(f'ERROR {e} Model {model} has no fts_idx column (has'
+                      f' upgrade step being executed?)')
+                success = False
+                continue
+
+            set_ftx = q.filter(model.fts_idx != None).count()
+            percentage = set_ftx / total * 100
+            if percentage < 60:
+                print(f'ERROR Percentage of indexed documents of model '
+                      f'{model} is too low: {percentage}% (has index step '
+                      f'being executed?)')
+                success = False
+            else:
+                print(f'Percentage of indexed documents of model '
+                      f'{model} is {percentage}%')
+
+        if not success:
+            print('\x1b[0;31;40m' + 'Indexing status check NOT OK' + '\x1b[0m')
+        else:
+            print('\x1b[0;32;40m' + 'Indexing status check OK' + '\x1b[0m')
```

Comment:
I would consider moving this entirely to the CLI command, so you can take advantage of `click.secho` for the styling and don't have to use `print`.